### PR TITLE
support id

### PR DESCRIPTION
**Pull request by zl910627**
id supports is needed

### CODE REVIEW

File: lib/src/vector_drawable.dart [hunk 1]
```diff
@@ -23,6 +23,8 @@ final Paint _grayscaleDstInPaint = Paint()
 /// Base interface for vector drawing.
 @immutable
 abstract class Drawable {
+  String get id;
```
Comment (dnfield) [inline lib/src/vector_drawable.dart:26]: Please add a docstring.

Probably something like

> A string that should uniquely identify this [Drawable] within its [DrawableRoot].

Is fine.

File: lib/src/vector_drawable.dart [hunk 2]
```diff
@@ -859,6 +865,9 @@ class DrawableRoot implements DrawableParent {
   @override
   final List<Drawable> children;
 
+  @override
+  String get id => 'root';
```
Comment (dnfield) [inline lib/src/vector_drawable.dart:869]: I think we should avoid this - someone might use that string in an SVG, and we should just take the `@id` attribute from the `<svg>` element itself instead.

File: lib/src/vector_drawable.dart (hunk 1 above)
Comment (zl910627) [reply lib/src/vector_drawable.dart:28]: Just added document as you suggested, thank you

**Comment by zl910627** [comment]
Hello @dnfield 
I have updated PR, and please check.
Looking forward updated package on Pub.
Thank you
Zheng

File: lib/src/vector_drawable.dart (hunk 2 above)
Comment (zl910627) [reply lib/src/vector_drawable.dart]: Yes, this has been fixed on new commit.
I also have added test case for this.
Thank you

**Pull request merged**